doc/nitrocli.1

```diff
@@ -401,6 +415,27 @@ see \fB\-\-no\-cache\fR).
 .TP
 .B NITROCLI_VERBOSITY
 Set the log level (integer, default: 0, see \fB\-\-verbose\fR).
+.SS Password & PIN entry
+The second set can be used to suppress interactive PIN entry through
+\fBpinentry\fR(1) or to provide a password to operations that otherwise would
+interactively ask for it. The following variables are recognized:
```

Comment:
It’s a bit hard to tell the difference betwen the two cases.  My first thought was pinentry vs. “interactively ask”, but we always use pinentry, right?  So it’s about PIN vs. password?

Reply:
Yeah, true. I've changed it to:
```
.SS Password & PIN entry
The second set can be used to provide password & PIN data to the program to
suppress interactive entry through \fBpinentry\fR(1) for operations that
otherwise would ask for it. The following variables are recognized:
```
Let me know if you can think of a better wording.

Reply:
Thanks!  I’d use “and” instead of “&”, but that’s just a personal preference.